src/Compilers/Core/Portable/CodeGen/VariableSlotAllocator.cs

```diff
@@ -69,17 +70,33 @@ internal abstract class VariableSlotAllocator
         /// <summary>
         /// Finds a closure in the previous generation that corresponds to the specified syntax.
         /// </summary>
+        /// <param name="closureSyntax">Syntax of the closure scope.</param>
+        /// <param name="parentClosureId">Id of the parent closure.</param>
+        /// <param name="structCaptures">Names of variables hoisted into the closure, or empty if the closure is a class.</param>
```

Comment:
Is `structCaptures` sometimes default and sometimes empty? Seems both are mentioned throughout the PR.

Reply:
Should say "default" here.